charts/base-cluster/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A generic, base cluster setup
 name: base-cluster
-version: 37.0.6
+version: 37.0.7
```

Comment:
you have removed a functionality of the base-cluster chart which is considered a breaking change.
The have to increase the major and not the patch version